app/views/devices_guidance/device_distribution_and_ownership.md

```diff
@@ -82,6 +81,11 @@ Organisations are responsible for deciding how they deliver devices. For example
 
 All distribution and return of lent devices should be done in accordance with [social distancing guidelines](https://www.gov.uk/government/publications/staying-alert-and-safe-social-distancing), and any [social distancing guidelines relevant to your local area](https://www.gov.uk/government/collections/local-restrictions-areas-with-an-outbreak-of-coronavirus-covid-19).
 
+### Redistributing unused devices
+
+Local authorities and academy trusts can use the support portal to see how much data is being used by each router they are responsible for. If you identify routers that are not being used, you should collect them and lend them to other children and young people who may have greater need.  The DfE is investigating options for extending data contracts on all 4G wireless routers that are being actively used.
```

Comment:
@gemmaemoore is it worth being more explicit about who "the support portal" is run by, or perhaps even link to it (or is that explained elsewhere?)